files/etc/radios.json

```diff
@@ -1236,6 +1236,24 @@
       "beamwidth": 6
     }
   },
+  "aruba ap-105": {
+    "wlan0": {
+      "maxpower": 24,
+      "antenna": {
+        "description": "2.1 dBi Omni",
+        "gain": 2.1,
```

Comment:
https://www.securewirelessworks.com/datasheets/Access-Points/DS_AP105.pdf

This lists different antenna gains than here. Is this correct?